TopMovies/DTO/MoviePreviewDTO.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  MoviePreviewDTO.swift
+//  TopMovies
+//
+//  Created by anikolaenko on 05.02.2021.
+//
+
+struct MoviePreviewDTO: Decodable {
+  let adult: Bool
+  let backdropPath: String?
+  let genreIds: [Int]
+  let id: Int
+  let originalTitle: String
+  let overview: String?
+  let popularity: Double
+  let posterPath: String?
+  let releaseDate, title: String
+  let video: Bool
+  let voteAverage: Double
+  let voteCount: Int
```

Comment:
Make sure that all of those fields are used in your lists and drop redundant if there're any.